staker/block_validator.go

```diff
@@ -174,7 +175,14 @@ func (c *BlockValidatorConfig) Validate() error {
 }
 
 type BlockValidatorDangerousConfig struct {
-	ResetBlockValidation bool `koanf:"reset-block-validation"`
+	ResetBlockValidation bool               `koanf:"reset-block-validation"`
+	Revalidation         RevalidationConfig `koanf:"revalidation"`
+}
+
+type RevalidationConfig struct {
+	StartBlock uint64 `koanf:"start-block"`
+	EndBlock   uint64 `koanf:"end-block"`
```

Comment:
It would be nice to have a validation somewhere checking that EndBlock >= StartBlock

Reply:
Added it now in BlockValidatorConfig's Validate